.github/workflows/build-keystone-operator.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
       uses: redhat-actions/openshift-tools-installer@v1
       with:
         source: github
-        operator-sdk: '1.18.1'
+        operator-sdk: '1.22.0'
```

Comment:
there is 1.22.1 , should go to that one, as we set in https://github.com/openstack-k8s-operators/install_yamls/pull/11 ?